packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ComponentsElement,
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import {
+  AnnotationElement,
+  toValue,
+  cloneShallow,
+  isObjectElement,
+} from '@swagger-api/apidom-core';
+import { isComponentsElement } from '@swagger-api/apidom-ns-openapi-3-0';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const isSecuritySchemeElement = (element: SecuritySchemeElement) => !!element; // TODO: Add import from @swagger-api/apidom-ns-openapi-3-1 when new release is released
+
+const securitySchemeTypeRefractorPlugin =
+  ({ annotations }: SecuritySchemeTypePluginOptions) =>
+  () => {
+    const removedSecuritySchemes: SecuritySchemeElement[] = [];
+    const annotation = new AnnotationElement(
+      '"mutualTLS" type is not supported in OpenAPI 3.0.3. It will be removed from the converted document.',
+      { classes: ['error'] },
+      { code: 'mutualTLS' },
+    );
+
+    return {
+      visitor: {
+        OpenApi3_1Element(element: OpenApi3_1Element) {
+          if (!isComponentsElement(element.components)) return undefined;
+          if (!isObjectElement(element.components.securitySchemes)) return undefined;
+
+          element.components.securitySchemes.forEach((securityScheme) => {
```

Comment:
```suggestion
          element.components.securitySchemes.forEach((value) => {
```

In ObjectElement context, the first argument here is a member `value` (second is key and third member), so it's better to call it that. In ArrayElement context the first argument is a current `item`.